sdk/tables/azure-data-tables/samples/async_samples/sample_create_delete_table_async.py

```diff
@@ -55,7 +64,7 @@ async def create_if_not_exists(self):
 
         # [START create_if_not_exists]
         async with TableServiceClient.from_connection_string(self.connection_string) as table_service_client:
-            table_item = TableServiceClient.create_table_if_not_exists(table_name=self.table_name)
+            table_item = await table_service_client.create_table_if_not_exists(table_name=self.table_name)
```

Comment:
consider using an async context manager so unclosed session warnings don't output to the console here (and in several other samples)